backend/cmm_helpers.mli

```diff
@@ -267,6 +267,8 @@ val unboxed_float_array_ref :
 val float_array_ref : expression -> expression -> Debuginfo.t -> expression
 val addr_array_set :
   expression -> expression -> expression -> Debuginfo.t -> expression
+val addr_array_set_local :
```

Comment:
Where is it used outside of `Cmm_helpers`?

Reply:
In Flambda 2.